include/repgp/repgp_def.h

```diff
@@ -94,8 +95,17 @@
 #define PGP_KEY_ID_SIZE 8
 
 /* Size of the fingerprint */
-#define PGP_FINGERPRINT_SIZE 20
-#define PGP_FINGERPRINT_HEX_SIZE (PGP_FINGERPRINT_SIZE * 2) + 1
+#define PGP_FINGERPRINT_V4_SIZE 20
+#if defined(ENABLE_CRYPTO_REFRESH)
+#define PGP_FINGERPRINT_V6_SIZE 32
+#define PGP_MAX_FINGERPRINT_SIZE PGP_FINGERPRINT_V6_SIZE
+#else
+#define PGP_MAX_FINGERPRINT_SIZE PGP_FINGERPRINT_V4_SIZE
+#endif
+#define PGP_MAX_FINGERPRINT_HEX_SIZE (PGP_MAX_FINGERPRINT_SIZE * 2) + 1
+
+/* SEIPDv2 salt length */
+#define PGP_SEIPDV2_SALT_LEN 32
```

Comment:
Please move it under the define.

Reply:
I don't really understand: move what exactly where?

Reply:
@falko-strenzke As `PGP_SEIPDV2_SALT_LEN` is used only with ENABLE_CRYPTO_REFRESH, it should be under the `#ifdef ENABLE_CRYPTO_REFRESH`, so code without `ENABLE_CRYPTO_REFRESH` doesn't have unused defines.

Reply:
addressed in upcoming commit